lib/themes/dosomething/paraneue_dosomething/templates/system/partials/navigation.tpl.php

```diff
@@ -28,20 +28,21 @@
 
     <ul class="navigation__secondary">
       <li>
-        <?php print $search_box; ?>
+	       <?php print $search_box; ?>
       </li>
       <?php if(!$logged_in): ?>
-      <?php // Will change 'Sign Up' to 'Create Account' once we refactor nav and can fit longer text! ?>
-      <?php if( theme_get_setting('show_profile_link') ): ?>
-      <li class="account"><a id="link--register" class="secondary-nav-item" href="<?php print $front_page; ?>user/register" data-modal-href="#modal--register"><?php print t('Register'); ?></a></li>
+        <?php if( theme_get_setting('show_profile_link') ): ?>
```

Comment:
I'm going to remove that conditional, Im confused how it managed to get back in there